packages/backend/src/clients/github/Github.ts

```diff
@@ -0,0 +1,167 @@
+import { NotFoundError } from '@lightdash/common';
+import { App, createNodeMiddleware } from '@octokit/app';
+import { createTokenAuth } from '@octokit/auth-token';
+import { Octokit, Octokit as OktokitRest } from '@octokit/rest';
+
+const { createAppAuth } = require('@octokit/auth-app');
+
+const privateKey = process.env.GITHUB_PRIVATE_KEY || '';
+const installationId = 47029382; // replace this once it is installed
+
+export const githubApp = new App({
+    appId: '703670',
+    privateKey,
+    oauth: {
+        clientId: process.env.GITHUB_CLIENT_ID || '',
+        clientSecret: process.env.GITHUB_CLIENT_SECRET || '',
+    },
+    webhooks: {
+        secret: 'Test',
```

Comment:
hardcoded secret on webhooks (mandatory field), we should make a new secret instead of using LIGHTDASH_SECRET